contracts/Fellowship.sol

```diff
@@ -191,18 +195,24 @@ contract Fellowship {
         emit NewWalkerInformation(msg.sender, _input, _output);
     }
 
-    function slashWalker(address _walker,uint256 _amount,bool _banish) external onlyRivendale {
+    // I guess we need to have a minimun amount of walkers. Otherwise current walkers could colude to progressively diminish the nuber of members until a small group is in full control

```

Comment:
You don't necessarily need it.  The walkers are only part of the vote for slashing, so theoretically the customers would want as many honest walkers as they can.  A minimum is also easy to get around since addresses can be the same people

Reply:
Good point. I guess this could be easily handled in future versions too